packages/uicore/src/hooks/useToaster/useToaster.tsx

```diff
@@ -21,26 +21,20 @@ export interface ToasterProps extends IToaster {
   showPrimary: (message: string | ReactNode, timeout?: number, key?: string) => void
 }
 
-const showToast = (options: IToastProps, key?: string): void => {
```

Comment:
reverting latest change made due to public access to hide toast with no error message.